h/buildext.py

```diff
@@ -35,13 +39,32 @@
 resolve = AssetResolver().resolve
 
 
-def build_extension_common(env, bundle_app=False):
+class Resolver(webassets.env.Resolver):
+    """A custom resolver for webassets which can resolve
+       package-relative paths (eg. 'h:path/to/asset.css') that are used
+       in assets.yaml
+    """
+
+    def search_for_source(self, ctx, item):
+        if item.startswith('../'):
+            # relative URLs in assets.yaml are relative to h/static
+            # Note that the return path here is not canonicalized
+            # (ie. '../' is not removed) for consistency with the way
+            # webassets resolves paths in the main app.
```

Comment:
Perhaps you could update this comment with the explanation you've given here. It's a lot clearer than the one currently in the source.